src/components/RouteError.jsx

```
@@ -0,0 +1,18 @@
+import { Link, useRouteError } from "react-router-dom";
+
+const RouteError = () => {
+  const error = useRouteError();
```

ESLint

[error] 'error' is assigned a value but never used.

(no-unused-vars)